pyam/iiasa.py

```diff
@@ -197,6 +217,10 @@ def _connection_map(self):
     @lru_cache()
     def valid_connections(self):
         """Return available resources (database API connections)"""
+        logger.warning(
+            "IIASA is migrating to a database infrastructure using the ixmp4 package."
```

Comment:
This could make a user think that all current projects running on the old infrastructure will be migrated.

Reply:
Fair point.